MIP16/mip16.md

```diff
@@ -58,9 +58,16 @@ It is important to note that due to the frequency of such changes, the Weekly Cy
 - A **Non-Standard Weekly Poll** is a non-binding weekly poll that has arbitrary time-sensitive decisions that MKR holders have to make in a relatively short period of time. These polls are needed to be expedited through the Maker governance process via a separate vote. The use of non-standard weekly polls is dedicated to Facilitators, given that they have already established a high level of trust with the community. Furthermore, the use of non-standard weekly governance polls will be limited to situations where the weekly governance cycle is determined to operate too slowly to be usable.
-- Facilitators that have been onboarded into [MIP38: DAO Primitives State](https://github.com/makerdao/mips/blob/master/MIP38/mip38.md) have been authorized by Maker Governance to efficiently interact with the weekly cycle to enable them to fulfill their Core Unit Mandate with as little friction as possible. Facilitators are authorized to submit non-standard Weekly polls that are related to their Core Unit Mandate and include logic in the weekly Executive Vote that is related to their Core Unit Mandate. If necessary, and if it is related to their Core Unit Mandate, Facilitators can skip the Non-Standard Weekly poll and include logic directly into the weekly Executive Vote.
 - To protect the integrity of existing MIPs, Signal Requests are only permitted when they are specifically triggered and required by an existing MIP.
 - This component explicitly forbids the possibility of community-triggered Signal Requests about arbitrary issues. It is also explicitly specified that the only way to make arbitrary changes to Maker Governance is through the full MIPs process or through Governance Mandates of Mandated Actors.
+- Facilitators that have been onboarded into [MIP38: Core Unit MIP Set Definitions](https://github.com/makerdao/mips/blob/master/MIP38/mip38.md) have been authorized by Maker Governance to efficiently interact with the weekly cycle to enable them to fulfill their Core Unit Mandate with as little friction as possible. Facilitators are authorized to submit non-standard Weekly polls that are related to their Core Unit Mandate and include logic in the weekly Executive Vote that is related to their Core Unit Mandate. If necessary, and if it is related to their Core Unit Mandate, Facilitators can skip the Non-Standard Weekly poll and include logic directly into the weekly Executive Votes.
+- Items will not be included in an executive vote until the following requirements have been met, this includes includes items proposed by Core Unit Facilitators:
+   - For code inclusions, smart contract code must be provided to PE-001 by the appropriate Core Unit or counterparty at least two weeks prior to the requested executive vote going live to ensure it can be properly reviewed.
```

Comment:
"at least two weeks prior to the requested executive vote going live"

This is the current cadence right?
Code delivered
Week 1: goerli work
Week2 mainnet - go live week.

Reply:
When were the contract addresses delivered for the Blocktower deal? Monday? that's 4 days before, which is the scenario we're trying to avoid

Reply:
hmm, then I think we need to discuss with Devs, cuz I would understand if other protocols or devs don't want to deploy mainnet smart contracts too early which can be costly and if the goerli executive requires some changes, then they need to redeploy the mainnet ones as well.
I would agree that goerli code and addresses must be delivered 2 weeks prior.

Reply:
useful feedback, can clarify that it's for Goerli, how long in advance is reasonable for mainnet in your opinion?